prover/protocol/distributed/compiler/plonkinwizard/distribute.go

```diff
@@ -0,0 +1,69 @@
+package plonkinwizard
+
+import (
+	"github.com/consensys/linea-monorepo/prover/protocol/distributed"
+	"github.com/consensys/linea-monorepo/prover/protocol/query"
+	"github.com/consensys/linea-monorepo/prover/protocol/wizard"
+	"github.com/consensys/linea-monorepo/prover/utils"
+)
+
+// DistributePlonkInWizard scans initComp for non-ignored PlonkInWizard queries and
+// identifies the ones corresponding to moduleName. For each of those queries, the
+// function inserts a corresponding query into the module. The function will panic
+// if:
+//
+//   - the query is cross-module
+//
+// The new query is analog to the one in initComp but and shares the same name.
+func DistributePlonkInWizard(
```

Comment:
Method defined, but in practice not called yet. I don't know the details of limitless prover, but its not yet fully functional?

Reply:
Also we are missing the big component to integrate everything, that's what I am working on. As far as it go, I cannot really confirm if this code will be integrated directly but I will use it as a draft.